rnote-ui/po/sl_SI.po

```diff
@@ -0,0 +1,2081 @@
+
+#: rnote-ui/data/ui/appmenu.ui:178
+msgid "Sidebar on the Right Side"
+msgstr ""
+
+#: rnote-ui/data/ui/appmenu.ui:188 rnote-ui/data/ui/shortcuts.ui:44
+msgid "Toggle Fullscreen"
+msgstr ""
+
+#: rnote-ui/data/ui/appwindow.ui:109
+msgid "Close the Folded Flap"
+msgstr ""
+
+#: rnote-ui/data/ui/appwindow.ui:126
+msgid "Workspace"
+msgstr "Delovni prostor"
+
+#: rnote-ui/data/ui/appwindow.ui:137
+msgid "Settings"
+msgstr "Nastavitve"
+
+#: rnote-ui/data/ui/canvasmenu.ui:24
+msgid "_Document Layout"
+msgstr "_Postavitev dokumenta"
+
+#: rnote-ui/data/ui/canvasmenu.ui:27
+msgid "_Fixed Size"
+msgstr "_Fiksna velikost"
+
+#: rnote-ui/data/ui/canvasmenu.ui:32
+msgid "_Continuous Vertical"
+msgstr ""
+
+#: rnote-ui/data/ui/canvasmenu.ui:37
+msgid "_Semi Infinite"
+msgstr ""
+
+#: rnote-ui/data/ui/canvasmenu.ui:42
+msgid "_Infinite"
+msgstr "_Neskončno"
+
+#: rnote-ui/data/ui/canvasmenu.ui:49
+msgid "Show _Format Borders"
+msgstr ""
+
+#: rnote-ui/data/ui/canvasmenu.ui:54
+msgid "Draw With _Touch Input"
+msgstr ""
+
+#: rnote-ui/data/ui/canvasmenu.ui:59
+msgid "Block Pinch to _Zoom"
+msgstr ""
+
+#: rnote-ui/data/ui/canvasmenu.ui:64
+msgid "_Pen Sounds"
+msgstr "_Zvok pisala"
+
+#: rnote-ui/data/ui/canvasmenu.ui:71
+msgid "C_lear Document"
+msgstr "_Sprazni dokument"
+
+#: rnote-ui/data/ui/canvasmenu.ui:75
+msgid "_Return to Origin Page"
+msgstr "_Vrnitev na izvorno stran"
+
+#: rnote-ui/data/ui/canvasmenu.ui:105 rnote-ui/data/ui/shortcuts.ui:120
+#: rnote-ui/data/ui/shortcuts.ui:126 rnote-ui/data/ui/shortcuts.ui:132
+msgid "Zoom out"
+msgstr "Oddalji"
+
+#: rnote-ui/data/ui/canvasmenu.ui:114
+msgid "Reset Zoom"
+msgstr "Ponastavi približevanje"
+
+#: rnote-ui/data/ui/canvasmenu.ui:121 rnote-ui/data/ui/shortcuts.ui:102
+#: rnote-ui/data/ui/shortcuts.ui:108 rnote-ui/data/ui/shortcuts.ui:114
+msgid "Zoom in"
+msgstr "Približaj"
+
+#: rnote-ui/data/ui/canvasmenu.ui:133
+msgid "Zoom to Page Width"
+msgstr "Povečaj na širino strani"
+
+#: rnote-ui/data/ui/canvasmenu.ui:148
+msgid "Remove Page"
+msgstr "Odstrani stran"
+
+#: rnote-ui/data/ui/canvasmenu.ui:156
+msgid "Add Page"
+msgstr "Dodaj stran"
+
+#: rnote-ui/data/ui/canvasmenu.ui:164
+msgid "Resize Document to Fit Strokes"
+msgstr ""
+
+#: rnote-ui/data/ui/colorpicker.ui:19
+msgid "Stroke Color"
+msgstr "Barva poteze"
+
+#: rnote-ui/data/ui/colorpicker.ui:26
+msgid "Fill Color"
+msgstr "Zapolni barvo"
+
+#: rnote-ui/data/ui/filerow.ui:58 rnote-ui/src/dialogs/import.rs:90
+msgid "Open"
+msgstr "Odpri"
+
+#: rnote-ui/data/ui/filerow.ui:62
+#: rnote-ui/src/workspacebrowser/filerow/actions/rename.rs:56
+msgid "Rename"
+msgstr "Preimenuj"
+
+#: rnote-ui/data/ui/filerow.ui:66
+msgid "Move to Trash"
+msgstr "Premakni v smeti"
+
+#: rnote-ui/data/ui/filerow.ui:70
+msgid "Duplicate"
+msgstr "Podvoji"
+
+#: rnote-ui/data/ui/mainheader.ui:19 rnote-ui/src/canvas/mod.rs:536
+msgid "Draft"
+msgstr "Osnutek"
+
+#: rnote-ui/data/ui/mainheader.ui:39 rnote-ui/data/ui/mainheader.ui:73
+msgid "Reveal/Hide Flap"
+msgstr ""
```

Comment:
todo: izboljšaj prevod in preveri konteks

```suggestion
msgstr "Razkrij/skrij loputo"
```

Reply:
Beseda "loputa" mi mogoče ni najbolj všeč, čeprav predstavlja neposredni prevod angl. besede _Flap_